eth-reference-tests/src/referenceTest/java/tech/pegasys/artemis/reference/phase0/epoch_processing/final_updates.java

```diff
@@ -24,22 +24,22 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
-import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.datastructures.state.BeaconStateImpl;
 import tech.pegasys.artemis.ethtests.TestSuite;
 import tech.pegasys.artemis.statetransition.util.EpochProcessorUtil;
 
 @ExtendWith(BouncyCastleExtension.class)
 public class final_updates extends TestSuite {
   @ParameterizedTest(name = "{index}. process final updates pre={0} -> post={1}")
   @MethodSource("mainnetFinalUpdatesSetup")
-  void mainnetProcessFinalUpdates(BeaconState pre, BeaconState post) throws Exception {
+  void mainnetProcessFinalUpdates(BeaconStateImpl pre, BeaconStateImpl post) throws Exception {
```

Comment:
looks like most of these reference test methods should be able to use the BeaconState interface types (`MutableBeaconState`)

Reply:
Yes, ref tests are still on the way